build.gradle.kts

```diff
@@ -34,7 +34,7 @@ plugins {
     idea
     kotlin("jvm") version "1.3.21"
     id("org.jetbrains.intellij") version "0.4.4"
-    id("org.jetbrains.grammarkit") version "2018.3.1"
+    id("org.jetbrains.grammarkit") version "2018.2.2"
```

Comment:
Why?

Reply:
@ice1000 this is a temporary solution to run CLion